README.md

```diff
@@ -88,12 +104,27 @@ jobs:
 
     steps:
       - name: Checkout repo
-        uses: actions/checkout@v2     
+        uses: actions/checkout@v2
+      # Obtain an AAD token and use it to run the notebook on Databricks
+      # Note: If running on AWS or GCP, you can directly pass your service principal
+      # token via the databricks-host input instead
+      - name: Log into Azure
+        uses: Azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+      # Get an AAD token for the SP using the programmatic ID for Azure Databricks as resourceId.
```

Comment:
```suggestion
      # Get an AAD token for the service principal, and store it in the DATABRICKS_TOKEN environment variable for use in subsequent steps. We set the `resource` parameter to the programmatic ID for Azure Databricks. See https://docs.microsoft.com/en-us/azure/databricks/dev-tools/api/latest/aad/service-prin-aad-token#--get-an-azure-ad-access-token for details.
```

Reply:
Would be nice to add a doc link to explain the magic string :) so people can confirm it's legit

Reply:
Done!